docs/mmteb/readme.md

```diff
@@ -0,0 +1,60 @@
+### 🔓 Task 4: Review PRs
+
+We welcome reviews of PRs adding new datasets. If you wish to review PRs of a specific language feel free to contact members of the MTEB team.
+
+## Authorship
+
+We follow a similar approach as in the [SeaCrowd Project](https://github.com/SEACrowd#contributing-to-seacrowd) and use a point-based system to determine co-authorships. 
+
+To be considered a co-author, at least 10 contribution points are required. Position of contributors in the author list is determined by the score they acquire, higher scores will appear first.
+
+To monitor how many points you have obtained, the contribution point tracking is now live at [this sheet](POINTS.md) and we recommend updating the score along with your PR. Past contributions also count. 
+
+Everyone with sufficient points will also be added to the MTEB GitHub and Huggingface repository as a contributor.
+
+# Contribution point guideline
+The contribution points are computed using the following table:
+
+> **Note**: The purpose of the point system is not to barrier collaboration, but to reward contributions. We might adjust the point requirement lower to accommodate more co-authorship if needed.
+
+
+| Contribution type   | Demand              | Points  | Description                                                                                                       |
+| ------------------- | ------------------- | ------- | ----------------------------------------------------------------------------------------------------------------- |
+| New dataset         | As many as possible | 2+bonus | Task 1. The First dataset for a language x task gains 4 bonus points. You can gain up to two bonuses pr. dataset. |
+| New task            | If relevant         | 10      | Task 2.                                                                                                           |
+| Dataset annotations | On demand           | 1       | Adding missing dataset annotations to existing datasets.                                                          |
+| (bug)fixes            | On demand           | 1-10    | Points depends the effect of code changes. If you want to find issues related to the MMTEB you can find them [here](https://github.com/embeddings-benchmark/mteb/milestone/1). |
+| Running Models      | On demand           | 1       | Task 3.                                   |
+| Review PR           | On demand           | 2       | Task 4.                                   |
+
+For the purpose of counting points, a language is defined by its [ISO 639-3](https://en.wikipedia.org/wiki/ISO_639-3) code, however, we encourage dialects or written language variants. All programming languages are considered one language.
+
+Team submissions are free to distribute points among the members as they like.
+
+## Communication Channels
+
+We will communicate via this GitHub repository. Please feel free to open issues or discussions and `Watch` the repository to be notified of any changes.
```

Comment:
Contextual kindly agreed to sponsor compute for this project. There will likely be missing scores at the end & some larger models we want to run, so this could be pretty useful I think. Let me know if not okay with you.

```suggestion
We will communicate via this GitHub repository. Please feel free to open issues or discussions and `Watch` the repository to be notified of any changes.

## Acknowledgements

We thank [Contextual AI](https://contextual.ai/) for sponsoring compute for this project.
```

Reply:
That's so great! Very nice of Contextual AI 😊

Reply:
Sounds great, think that is a great idea